gdk4/src/auto/device.rs

```diff
@@ -118,22 +118,8 @@ impl Device {
         unsafe { from_glib(ffi::gdk_device_has_bidi_layouts(self.to_glib_none().0)) }
     }
 
-    pub fn get_property_has_bidi_layouts(&self) -> bool {
```

Comment:
This one got removed as well

Reply:
Same as for `is_local`. See https://github.com/gtk-rs/gtk4-rs/blob/8107dfccc9c8a3939f1eaf45b23911dfc05d986e/gdk4/src/auto/device.rs#L117.